processor/batchprocessor/README.md

```diff
@@ -15,8 +15,8 @@ any data drops such as sampling.
 Please refer to [config.go](./config.go) for the config spec.
 
 The following configuration options can be modified:
-- `send_batch_size` (default = 8192): Number of spans or metrics after which a
-batch will be sent regardless of the timeout.
+- `send_batch_size` (default = 8192): Number of spans, metric data points, or log
+records in metrics after which a batch will be sent regardless of the timeout.
```

Comment:
Please document the split logic, since it is not a trivial split of data-points, actually you split at metric level but ensure data points max (which I think is the right thing to do).

Reply:
Done, wdyt about this version?

Reply:
@bogdandrutu friendly ping